viser/_message_api.py

```diff
@@ -70,6 +71,45 @@ def _colors_to_uint8(colors: onp.ndarray) -> onpt.NDArray[onp.uint8]:
 ]
 
 
+class TitlebarButton(dict):
```

Comment:
- `typing.TypedDict` is probably a good alternative for dictionary types like this
- a dataclass could also work?